site/package.json

```diff
@@ -3,7 +3,6 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "babel-plugin-preval": "^1.6.4",
     "babel-plugin-styled-components": "^1.5.1",
     "markdown-to-jsx": "^6.5.2",
     "next": "^5.0.0",
```

Comment:
Doesn’t look like v6 was upgraded here? That’s probably why the PR Netlify deploy is failing.

Reply:
Yep, for some reason `yarn` crashes on node-gyp 🤔